nautobot/ipam/templates/ipam/vlangroup.html

```diff
@@ -1,34 +1,8 @@
-{% extends 'base.html' %}
-{% load buttons %}
```

Comment:
From the CI failure, we still need to `load buttons` since we're overriding the `buttons` block below.